web/themes/custom/cloudy2/templates/media/media--document--search-result.html.twig

```diff
@@ -52,3 +54,4 @@
   {% endif %}
 </div>
 
+console.log(mime_type)
```

Comment:
This file is a bit of a mess. 7 months ago, at least I assume I did the PR review, I missed a pretty significant error that introduced a duplicate H2 element in this display mode.

Then the console.log(mime_type) got left in the file in this PR. 😞